modules/standard/IO.chpl

```diff
@@ -2688,23 +2768,33 @@ This function is equivalent to calling :proc:`open` and then
           channel should no longer be allowed to read. Defaults
           to a ``max(int(64))`` - meaning no end point.
 :arg hints: optional argument to specify any hints to the I/O system about
-            this file. See :type:`iohints`.
+            this file. See :record:`ioHintSet`.
 :returns: an open reading channel to the requested resource.
 
 :throws SystemError: Thrown if a reading channel could not be returned.
  */
 proc openreader(path:string,
                 param kind=iokind.dynamic, param locking=true,
                 start:int(64) = 0, end:int(64) = max(int(64)),
-                hints:iohints = IOHINT_NONE)
+                hints=ioHintSet.empty)
     : channel(false, kind, locking) throws {
   return openreaderHelper(path, kind, locking, start, end, hints);
 }
 
+pragma "last resort"
+deprecated "The 'iohints' type is deprecated; please use a variant of 'openreader' that takes an 'ioHintSet' instead."
+proc openreader(path:string,
```

Comment:
There should be tests of these deprecated functions, too, sorry for not thinking of it sooner